app/controllers/AuthenticationController.scala

```diff
@@ -429,6 +431,91 @@ class AuthenticationController @Inject()(
     }
   }
 
+  private lazy val oidcClientId = conf.WebKnossos.OIDC.clientId
+  private lazy val oidcProviderUrl = conf.WebKnossos.OIDC.providerURL
```

Comment:
since these have only one usage and the readability should still be ok, I’d inline them below